src/datasources/tag/README.md

```diff
@@ -0,0 +1,6 @@
+# Systemlink Tag data source
+
+<!--
+    TODO: Write a short description of your plugin and document any extra
+    configuration that is required for development.
+-->
```

Comment:
Can you fill this out? You can see an example in Grace's PR: https://github.com/ni/systemlink-grafana-plugins/pull/3